mage_ai/io/postgres.py

```diff
@@ -73,14 +88,19 @@ def load(self, query_string: str, limit: int = QUERY_ROW_LIMIT, **kwargs) -> Dat
             return read_sql(self._enforce_limit(query_string, limit), self.conn, **kwargs)
 
     def export(
-        self, df: DataFrame, name: str, index: bool = False, if_exists: str = 'replace', **kwargs
+        self,
+        df: DataFrame,
+        table_name: str,
```

Comment:
I think you also need to specify schema_name to identify a table.

Reply:
Forgot about schemas, added `schema_name` parameter + auto creation of schema if the given schema doesn't exist.